web/modules/custom/portland/src/Plugin/Block/RevisionBlock.php

```diff
@@ -21,8 +21,8 @@
 class RevisionBlock extends BlockBase {
     private static $severity_icon = [
       'danger' => 'ban',
-      'warning' => 'exclamation-circle',
-      'success' => 'info-circle'
+      'warning' => 'exclamation-triangle',
```

Comment:
I'm actually okay with exclamation-circle in this case. The triangle is more of a warning where the circle is more of a notice? I would totally approve this either way, but wanted to get this into place.

Reply:
I hear what you're saying. I'm going to create a message component story so we can then swap out message instances for a component. I'll make a note to revisit the icon choice/intention, warning vs. notice usage.